examples/sundials/CMakeLists.txt

```diff
@@ -51,7 +51,10 @@ if (MFEM_ENABLE_TESTING)
   set(EX10_COMMON_OPTS -m ../../data/beam-quad.mesh -o 2 -s 5 -dt 0.15 -tf 6 -vs 10)
   set(EX10_TEST_OPTS ${EX10_COMMON_OPTS}  -r  2)
   set(EX10P_TEST_OPTS ${EX10_COMMON_OPTS} -rp 1)
-  # Example 16: use the default options
+  # Example 16: test ARKODE with implicit time stepping using mass form
+  set(EX16_COMMON_OPTS -s 15)
+  set(EX16_TEST_OPTS ${EX16_COMMON_OPTS})
+  set(EX16P_TEST_OPTS ${EX16_COMMON_OPTS})
```

Comment:
This modification should be applied to the `makefile` as well -- we want the same tests to be run by both GNU make and CMake. If you want to have two tests (e.g., one without `-s 15` and one with it) then we should have them both in both build systems.

Reply:
Good point: this should now be done!